src/snyk/common/configuration/configuration.ts

```diff
@@ -17,6 +17,7 @@ import {
   YES_TELEMETRY_SETTING,
   YES_WELCOME_NOTIFICATION_SETTING,
 } from '../constants/settings';
+import { ExtensionContext } from '../vscode/extensionContext';
```

Comment:
Looks like this is unnecessary here?

Reply:
👍 you're right. It doesn't hurt since no impact on runtime performance but I've removed it to have it cleaner :-)

Reply:
Also like that you keep an eye on GH Actions warning (tslint) 👌 let's reduce the amount of them in the code 💯